docs/development/i18n.rst

```diff
@@ -92,43 +100,58 @@ new language is completely translated and reviewed, the
-must also be updated. Individual developers should not do this
-whenever changing strings in the code; the translations are updated in
-bulk when it's time to update the Weblate fork.
-
-Translations can be updated with the following command:
-
-.. code:: sh
-
-    $ make translate
-
-This wraps ``i18n_tool.py translate-messages`` and ``i18n_tool.py
-translate-desktop``.  These commands update the `.pot`_ files for the
-SecureDrop server code and the desktop icons, as well as the `.po`_
-files for each language.
-
-.. note:: The new source strings will only be visible to translators
-    in `Weblate`_ after the ``develop`` branch is merged into the
-    Weblate fork.
+Whenever strings are modified in the SecureDrop source code, templates
+or desktop icon labels, the translation files should also be updated
+by running ``make translate`` in the root of the SecureDrop working
+copy.
+
+The ``translate`` target runs ``i18n_tool.py translate-messages`` and
+``i18n_tool.py translate-desktop``, which in turn use ``pybabel
+extract`` to gather source strings.  These commands update the `.pot`_
+files for the SecureDrop server code and the desktop icons, as well as
+the `.po`_ files for each language.
+
+After running ``make translate``, carefully review the output of ``git
+diff``. Check ``securedrop/messages.pot`` first for updated strings,
+looking for problems like:
+
+* overly idiomatic English
+* fragmented text, such as pieces of a sentence intended to be
+  concatenated together, which can be difficult to translate
+* messages that are marked with plain ``gettext`` and contain plurals
+  based on numeric placeholder variables -- these should generally be
+  marked with ``ngettext`` so that they can be translated properly in
+  languages with complex plural forms
+
+Then review the ``messages.po`` of one existing translation, with a
+focus on new translations, which are often marked ``fuzzy``. There is
+no need to review multiple languages' ``.po`` files because they are
+processed in the same way.
+
+Once you've reviewed the changes, submit them in a pull request for
+the ``develop`` branch in the `SecureDrop git repository`_.
+
+The new source strings will only be visible to translators in
+`Weblate`_ after they've been merged to ``develop`` and that branch
+has been merged into the Weblate fork. The localization manager does
+this at the beginning of our release cycle.
 
 .. _merge_develop_to_weblate:
 
 Merge develop into the Weblate fork
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-`Weblate`_ works on a long standing fork of the `SecureDrop git
-repository`_ and is exclusively responsible for the content of the
-``*.pot`` and ``*.po`` files. The content of the ``develop`` branch
-must be merged into the ``i18n`` branch to make updated source strings
-available to Weblate.
+1) First make sure the translation files on ``develop`` contain the
+latest source strings. Follow the steps under
+:ref:`update_strings_to_be_translated`.
```

Comment:
this appears as `:ref:`update_strings_to_be_translated`.` in the formatted doc

Reply:
I'm not seeing that:
![update-strings-ref](https://user-images.githubusercontent.com/47430598/111365723-368f9680-8669-11eb-9f3e-74cead64d6f9.png)

Reply:
whoops my bad